src/app/components/task/GeolocationTaskResponse.js

```diff
@@ -53,40 +64,33 @@ class GeolocationTaskResponse extends Component {
       <FlexRow className="task__geolocation-response">
         <span className="task__response"><ParsedText text={name} /></span>
         {coordinatesString ?
-          <span
+          <Box component="span"
```

Comment:
`67:16  error  Property should be placed on a new line  react/jsx-first-prop-new-line`